app/models/financialDetails/ChargeItem.scala

```diff
@@ -80,6 +80,13 @@ case class ChargeItem (
     subTransactionType.exists(subType => codingOutSubTypes.contains(subType))
   }
 
+  def notCodedOutPoa: Boolean = {
```

Comment:
Please move this method from [ChargeItem] and [TaxYearSummaryChargeItem] to base trait => TransactionItem
here is example:
https://github.com/hmrc/income-tax-view-change-frontend/pull/2567/files